### ISSUE TITLE
Build failed with Xcode 15.0 beta 5

### ISSUE THREAD

**Issue by nixzhu** [firebase/firebase-ios-sdk#11608]
### Description

DT_TOOLCHAIN_DIR cannot be used to evaluate LIBRARY_SEARCH_PATHS, use TOOLCHAIN_DIR instead

### Firebase SDK Version

10.12.0

### Xcode Version

15.0 beta 5

### Installation Method

CocoaPods

### Firebase Product(s)

Analytics

### Targeted Platforms

iOS

### Relevant Log Output

```shell
DT_TOOLCHAIN_DIR cannot be used to evaluate LIBRARY_SEARCH_PATHS, use TOOLCHAIN_DIR instead
```

**Comment by ninhgeocomply** [comment]
I have the same issues too. Firebase version 9.4.0.
<img width="314" alt="Screenshot 2023-07-26 at 11 29 13" src="https://github.com/firebase/firebase-ios-sdk/assets/93517594/bd56afc9-d738-461c-a921-5733f397510e">

**Comment by marcuswu0814** [comment]
It seems due to a CocoaPods problem, maybe can flow this [PR](https://github.com/CocoaPods/CocoaPods/pull/12009) and try it with [doc](https://guides.cocoapods.org/using/unreleased-features).

**Comment by Norod** [comment]
> It seems due to a CocoaPods problem, maybe can follow this [PR](https://github.com/CocoaPods/CocoaPods/pull/12009) and try it with [doc](https://guides.cocoapods.org/using/unreleased-features).

Thank you for this. Works like magic, hope it will get merged soon.

**Comment by paulb777** [comment]
Closing here since it is a CocoaPods issue that looks to be planned to be fixed in CocoaPods 1.13.0. Please follow https://github.com/CocoaPods/CocoaPods/pull/12009 for status and workarounds.
